docs/boot.md

```diff
@@ -0,0 +1,48 @@
+## There is a more detailed explanation in the [circuit python docs](https://docs.circuitpython.org/en/latest/README.html), however there are some common use cases for your keyboard listed here
+
+### You can hide your device from showing up as a usb storage
+
+```python
+storage.disable_usb_drive()
+```
+
+### Make your keyboard work in bios
+
+```python
+usb_hid.enable(boot_device=1)
+```
+
+### Disable serial comms
+
+```python
+# Equivalent to usb_cdc.enable(console=False, data=False)
+usb_cdc.disable()
+```
+
+### A fully working example, with a
+
+```python
+import supervisor
+import board
+import digitalio
+import storage
+import usb_cdc
+import usb_hid
+
+supervisor.set_next_stack_limit(4096 + 4096)
```

Comment:
A comment to this line would be nice, like "this is the circuit python default" or "refer to your board specific boot.py". I don't know if this is board specific.

Reply:
Comment added.